server/engine/src/test/java/org/eclipse/lsp/cobol/lsp/SourceUnitGraphTest.java

```diff
@@ -103,10 +103,10 @@ void testLinksBetweenCopybookAndSourceCode() {
         AnalysisState.COMPLETED, model, SourceUnitGraph.EventSource.FILE_SYSTEM);
 
     assertEquals("COPY 3 TEXT", sourceUnitGraph.getCopyNodeContent(copyNode3));
-    assertTrue(sourceUnitGraph.isCopybook(copy1Uri));
-    assertTrue(sourceUnitGraph.isCopybook(copy2Uri));
-    assertTrue(sourceUnitGraph.isCopybook(copy3Uri));
-    assertFalse(sourceUnitGraph.isCopybook(URI));
+    assertTrue(sourceUnitGraph.isUserSuppliedCopybook(copy1Uri));
+    assertTrue(sourceUnitGraph.isUserSuppliedCopybook(copy2Uri));
+    assertTrue(sourceUnitGraph.isUserSuppliedCopybook(copy3Uri));
+    assertFalse(sourceUnitGraph.isUserSuppliedCopybook(URI));
```

Comment:
These changes looks like are from 8bd8a7135735dd185a3a0a6aebd042b6e19260fb , and unrelate to this PR